packages/components/addon/components/hds/interactive/index.js

```diff
@@ -22,4 +23,11 @@ export default class HdsInteractiveComponent extends Component {
   get isRouteExternal() {
     return this.args.isRouteExternal ?? false;
   }
+
+  @action
+  onKeyUp(event) {
+    if (event.key === ' ') {
```

Comment:
[question] I usually use `event.code === 'Space' || event.key === 'Space'` - do you have confidence that the approach used here will work for all keyboard types?